pandas/core/computation/expressions.py

```diff
@@ -107,15 +107,12 @@ def _evaluate_numexpr(op, op_str, a, b, reversed=False):
 
         a_value = getattr(a, "values", a)
         b_value = getattr(b, "values", b)
-        try:
-            result = ne.evaluate(
-                "a_value {op} b_value".format(op=op_str),
-                local_dict={"a_value": a_value, "b_value": b_value},
-                casting="safe",
-            )
-        except ValueError as detail:
-            if "unknown type object" in str(detail):
-                pass
```

Comment:
This did never happen?

Reply:
the only errors that bubble up are TypeErrors, which we catch in the calling func in ops.array_ops